src/main/java/de/samply/share/client/util/connector/CentralSearchConnector.java

```diff
@@ -256,14 +260,38 @@ public DateRestriction getDateRestriction() throws CentralSearchConnectorExcepti
     }
   }
 
+  private UploadStats deserialize(String stringUploadStats)
+      throws IOException, ClassNotFoundException {
+
+    byte[] byteArrayUploadStats = Base64.getDecoder().decode(stringUploadStats);
+    InputStream inputStream = new ByteArrayInputStream(byteArrayUploadStats);
+    ObjectInputStream objectInputStream = new ObjectInputStream(inputStream);
+    UploadStats uploadStats = (UploadStats) objectInputStream.readObject();
```

Comment:
This is Java Serialisation and not XML. Please use Jackson here.

Reply:
Implemented